src/dev/ce-migration/index.md

```diff
@@ -3,5 +3,115 @@
 
 Migration guides are here to help you migrate to the latest version of [<MainPlatformName /> Community Edition](../mergince/).
 
-## Migration guide to 2023.2
+## Migration guide from any previous version to 2023.2
 
+Besides various fixes, enhancements and performance improvements the most notable change recently introduced is the concept of workspaces. For Community Edition it means there is a **common shared workspace (global workspace)** for all users where all projects are stored, instead of having a personal or organizational namespace for projects.
+
+:::tip
+In case you do not need your previous data, we advise to start with clean deployment without the need to follow this migration guide.
+:::
+
+**Upgrading to 2023.2**
+
+ 1. Sync all your projects. Hit synchronise on all devices that have local changes (mobiles, QGIS plugins)
+
+ 2. Remove downloaded projects from all devices (mobiles, QGIS plugins)
+
+ 3. Backup your database ⚠️
+
+```bash
+$ docker exec mergin-db pg_dump -U postgres -Fc postgres > pg_backup.dump
+```
+
+ 4. Stop all running Mergin Maps services (from project root folder)
+```bash
+$ docker-compose -f docker-compose.yml stop
+```
+
+ 5. Pull the latest changes
+```bash
+$ git pull
+```
+
+ 6. Amend environment variables (.env file). **Important** ⚠️
```

Comment:
We should probably link steps here to quick start guide. After pulling changes from new master branch, next step is to run new mergin maps (via docker-compose)

Reply:
Isn't it the step 7?

Reply:
maybe we should just replace step 6 and 7 with link to quick start?

Reply:
Hmm, but wouldn't "quick start" put their DB into the latest migration state? How would one then run DB migrations?

Reply:
true, they must not run init-db command ... not sure, but step 6 should be done via quick start guide as it is really production setup